app/javascript/controllers/single_playback_controller.js

```diff
@@ -23,10 +23,12 @@ export default class extends PlaybackController {
     if (!this.loaded) {
       this.bigPlay.animation.showPause()
       this.loaded = true
+      this.bigPlay.progressContainerInnerTarget.classList.add('visible')
     }
     this.bigPlay.percentPlayed = this.percentPlayed()
     this.bigPlay.waveform.update()
     this.bigPlay.timeTarget.innerHTML = this.time
+    this.bigPlay.progressContainerInnerTarget.style.left = 100 * this.percentPlayed() + "%"
```

Comment:
@ofsound Since these 4 lines in this `if` condition will have to also be in `playlist_playback_controller.js` I was thinking we could instead call `this.bigPlay.update(this.percentPlayed())`

The `update` method could look something like this:

```javascript
update(percentPlayed) {
  this.percentPlayed = percentPlayed
  waveform.update
  this.timeTarget.innerHTML = this.delegate.time
  this.progressContainerInnerTarget.style.left = 100 * this.percentPlayed() + "%"
}
```